source/errors/404.html.md.erb

```diff
@@ -0,0 +1,14 @@
+---
+title: 404 Page not found
```

Comment:
```suggestion
title: Page not found
```

Should just be 'Page not found', as per the page not found pattern.

Reply:
This matches the Design System [404 page title](https://github.com/alphagov/govuk-design-system/blob/311350fa50bce0a087760ba49a0e38885f0f5550/src/errors/404.njk#L2). Should we change that as well? It does sound a bit odd.

Reply:
Yep, I reckon so 👍 The pattern in the Design System says 'Do not use […] technical jargon like 404 or bad request'.

Reply:
https://github.com/alphagov/govuk-design-system/pull/1232